$OpenBSD: patch-config_h_in,v 1.7 2014/05/28 08:04:47 ajacoutot Exp $

XXX push upstream
Use arc4random_stir only if present.

--- config.h.in.orig	Thu Feb 27 16:57:59 2014
+++ config.h.in	Wed May 28 09:00:35 2014
@@ -612,7 +612,11 @@
 
 #ifdef HAVE_ARC4RANDOM
 #  define CUPS_RAND() arc4random()
-#  define CUPS_SRAND(v) arc4random_stir()
+#  ifdef HAVE_ARC4RANDOM_STIR
+#    define CUPS_SRAND(v) arc4random_stir()
+#  else
+#    define CUPS_SRAND(v)
+#  endif
 #elif defined(HAVE_RANDOM)
 #  define CUPS_RAND() random()
 #  define CUPS_SRAND(v) srandom(v)
